GitCommands/Git/GitVersion.cs

```diff
@@ -176,7 +176,11 @@ private static int Compare(GitVersion left, GitVersion right)
 
         public override string ToString()
         {
-            return Full.Replace(".msysgit.0", "").Replace(".msysgit.1", "");
+            return Full
+                .Replace(".msysgit.0", "")
+                .Replace(".msysgit.1", "")
+                .Replace(".windows.0", "")
+                .Replace(".windows.1", "");
```

Comment:
you're right, i must admit i didn't feel like making the code more flexible than it was before.